src/main/java/edu/kis/powp/jobs2d/drivers/DriverCommands/OperateToCommand.java

```diff
@@ -0,0 +1,22 @@
+package edu.kis.powp.jobs2d.drivers.DriverCommands;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class OperateToCommand implements DriverCommand{
+
+    private int x, y;
+    private Job2dDriver driver;
+
+    public OperateToCommand(int x, int y, Job2dDriver driver)
```

Comment:
The solution lacks flexibility, using different drivers with commands is difficult (e.g., you could consider setter).